src/ChangeLog.txt

```diff
@@ -2557,17 +2557,4 @@ v0.91.4.10:
     - Fix mayors being able to be made into Outlaws.
     - Fix an issue with plotManagementRevertSpeed on mysql databases.
     - Fix issue plaguing Mysql DB's in 0.91.4.9 where the console was spammed endlessly by revert speed default settings.
-    - Fix issue with outposts not unclaim their outpostspawns.
-v0.91.4.11:
-    - Fix 'Format specifier "%s"' error seen when attempting to add a town to you nation that doesn't have the config's required_number_residents_join_nation.
```

Comment:
Funky Merge, Removed and Re-Added lines. In the file it still exists. Happened again.